be/src/runtime/memory/cache_policy.h

```diff
@@ -0,0 +1,133 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#pragma once
+
+#include "olap/lru_cache.h"
+#include "util/stopwatch.hpp"
+#include "util/time.h"
+
+namespace doris {
+
+static constexpr int32_t CACHE_GC_MIN_FREE_SIZE = 67108864; // 64M
+
+template <typename T>
+class CachePolicy {
```

Comment:
add detail comments for this class, so that other developer could extend from it.

Reply:
ok, I will review the code twice, fix ut, and add comments